### PR TITLE
Add override functionality to links, allowing to customise their wrapper

### PR DESCRIPTION
**Pull request by gfaugere**
Just added a `wrapper` in the Link class, keeping the same default one.

Let me know what you think 😄

### CODE REVIEW

File: src/listener/Link.php [hunk 1]
```diff
@@ -14,14 +14,17 @@
  */
 class Link extends InlineListener
 {
+
+    public $wrapper = '<a href="{link}" target="_blank">{text}</a>';
```
Comment (nadar) [inline src/listener/Link.php]: please add php doc including @since tag.

Comment (gfaugere) [reply src/listener/Link.php]: Done! Not sure what else to say there 😕

Comment (nadar) [reply src/listener/Link.php]: Take a look at the example i just made: https://github.com/nadar/quill-delta-parser/pull/29/files

Comment (gfaugere) [reply src/listener/Link.php]: Alright, updated!

**Pull request merged**